library/src/main/java/com/bumptech/glide/load/model/ModelEquity.java

```diff
@@ -0,0 +1,16 @@
+package com.bumptech.glide.load.model;
+
+/**
+ * An optional interface that models can implement to handle difference in equity
+ * between the model and the target.
+ */
+public interface ModelEquity {
+
+    /**
+     * Returns {@code true} if this Model is equivalent to the given Model.
+     *
+     * @param other the model to be compared to
```

Comment:
you can drop these two comments since you're describing them already in your summary statement above.